index.rst

```diff
@@ -460,7 +441,85 @@ to be accessible from Kafka, and a slow consumer can have additional processing
+on the producer end.
+
++-----------------------------------------------+--------------------+------------------------+-------+
+|     Experiment                                |Producing Time (s)  | Consuming Time (s)     | Total |
++===============================================+====================+========================+=======+
+| 100 producers, 11 consumers, 10 partitions    |3.75 +/- 1.30       | 10.8 +/- 3.52          | 14.55 |
++-----------------------------------------------+--------------------+------------------------+-------+
+| 10 producers, 2 consumers, 1 partition        |7.46 +/- 0.23       | 11.0  +/- 2.0          | 18.46 |
++-----------------------------------------------+--------------------+------------------------+-------+
+| 100 producers, 2 consumers, 1 partition       |5.9 +/- 1.3         | 14.9 +/- 1.3           | 20.8  |
++-----------------------------------------------+--------------------+------------------------+-------+
+| 10 producers, 4 consumers, 1 partition        |6.26 +/- 0.93       | 17.91 +/- 3.33         | 24.17 |
++-----------------------------------------------+--------------------+------------------------+-------+
+| 10 producers, 6 consumers, 1 partition        |7.15 +/- 1.34       | 21.14 +/- 2.17         | 28.29 |
++-----------------------------------------------+--------------------+------------------------+-------+
+| 1 producer, 2 consumers, 1 partition          |33.1 +/- 2.02       | 0.16 +/- 0.23          | 33.26 |
++-----------------------------------------------+--------------------+------------------------+-------+
+| 10 producers, 11 consumers, 1 partition       |8.82 +/- 2.41       | 2068 +/- 1131          | 2077  |
++-----------------------------------------------+--------------------+------------------------+-------+
+
+
+Conclusions
+-----------
+
+TO BE EXTENDED AND ORGANIZED IN TICKET DM-13307.
+
+For most experiments and metrics measured, assuming that the size of alerts has been estimated accurately,
+there is a significant impact on Kafka compute utilization and
+timing when postage stamp cutouts are included in the alert packets.
+The inclusion of postage stamp cutouts is currently a science requirement, however,
+results here suggest that additional consideration be
+given to alternatives for the distribution of stamps, if it is possible
+that the current requirements may be adjusted.  One alternative to including postage
+stamps in the stream, recommended by other users of Kafka for large messages, is to forego including the
+stamps in the stream and instead include a url to the location of the stamps from which the stamp can be
+accessed separately.  This would particularly alleviate the increased network traffic for each additional
+consumer reading a stream.  Another potentially possible alternative could be to separate the postage stamps
+into their own Kafka topic.  If not all primary consumers are required or expected to read postage stamp
```

Comment:
Split the text after the last comma into its own sentence.

Reply:
Ok, fixed.